src/main/java/de/tum/in/www1/artemis/domain/plagiarism/PlagiarismSubmission.java

```diff
@@ -45,9 +47,8 @@
     /**
      * List of elements the related submission consists of.
      */
-    @OneToMany(cascade = CascadeType.ALL, targetEntity = PlagiarismSubmissionElement.class, fetch = FetchType.LAZY)
-    // TODO: we don't need a join table here and should simply store the 'submissionId' in the element
-    @JoinTable(name = "plagiarism_submission_elements", joinColumns = @JoinColumn(name = "plagiarism_submission_id"), inverseJoinColumns = @JoinColumn(name = "plagiarism_submission_element_id"))
+    @JsonIgnoreProperties("plagiarismSubmission")
+    @OneToMany(mappedBy = "plagiarismSubmission", cascade = CascadeType.ALL, targetEntity = PlagiarismSubmissionElement.class, fetch = FetchType.LAZY)
```

Comment:
fetchType.Lazy is default setting

Reply:
We added it there and I left it there on purpose since this was the main issue causing the server to crash when the elements were loaded. This is just there to make it absolutely clear that this should be lazy loaded, but I can also remove it if you prefer it that way